examples/mixture_of_experts/dist_moe.py

```diff
@@ -0,0 +1,96 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates
+# torchrun --nproc-per-node 5 dist_moe.py
```

Comment:
I think we should keep the standard format and preface the run instructions as usual:
"# Minimum effort to run this example:"
I actually missed this line b/c of it and had to work backwards until I figured out it was expecting n_experts+2 for world size.